docs-archive/apache-airflow-providers/_modules/index.html

```diff
@@ -29,8 +29,8 @@
     
 
 <header>
-
-    <nav class="js-navbar-scroll navbar">
+    <a href="https://airflowsummit.org" target="_blank" class="d-block fixed-top px-3 py-2 bg-success text-center text-bold text-white">Airflow Summit 2021 is coming July 8-16. Register now!</a>
```

Comment:
Maybe we should remove that one:)

Reply:
Oh yeah let me remove that on entire website in one-go